ip_email_record.js

```diff
@@ -7,11 +7,15 @@ module.exports = function (RATE_LIMIT_INTERVAL_MS, MAX_BAD_LOGINS, now) {
 
   now = now || Date.now
 
-  var IP_EMAIL_ACTIONS = [
-    'accountLogin',
-    'accountDestroy',
-    'passwordChange'
-  ]
+  var IP_EMAIL_ACTION = {
+    accountLogin   : true,
+    accountDestroy : true,
+    passwordChange : true,
+  }
+
+  function isIpEmailAction(action) {
+    return IP_EMAIL_ACTION[action]
+  }
```

Comment:
Aha, I see, these are actions that accept authPW as input and hence allow guessing attempts on the user's password.